decision/policies.py

```diff
@@ -36,3 +37,32 @@ def get_best_action(world: World):
                 best_action = action
 
         return best_action
+
+
+class SwapAllPolicy(Policy):
+    @staticmethod
+    def get_best_action(world: World):
+        # Choose a random cluster
+        next_cluster: Cluster = random.choice(
+            [
+                cluster
+                for cluster in world.state.clusters
+                if cluster.id != world.state.current_cluster.id
+            ]
+        )
```

Comment:
To make this swap policy better, choose the cluster that is has the largest deviation from ideal state (something like: sorted(world.state.clusters, lambda cluster: cluster.ideal_state - cluster.total_battery)

Reply:
We have discussed this, and this will be implemented in PRs coming in the near future.